astroquery/nasa_exoplanet_archive/core.py

```diff
@@ -420,18 +496,13 @@ def _fix_units(self, data):
                 else:  # pragma: nocover
                     warnings.warn("Unrecognized unit: '{0}'".format(unit), AstropyWarning)
 
-            # Here we're figuring out out if the column is masked because this doesn't
-            # play nice with quantities so we need to keep track of the mask separately.
+            # Unmask since astropy doesn't like masked values in columns with units
```

Comment:
😱 really?!  that's annoying.  (you can mark this resolved, I just had to remark on it)